docs/source/getting_started/user_guide.rst

```diff
@@ -0,0 +1,47 @@
+.. _user_guide:
+
+User Guide
+==========
```

Comment:
Rather than user guide I would use the term Developer's Guide (from what appears in the guide).

Reply:
Perhaps a user guide could be screenshots of each step of the process to load datasets and run experiments.